Microwave.Classes/Controllers/UserInterface.cs

```diff
@@ -27,7 +28,8 @@ private enum States
             IDoor door,
             IDisplay display,
             ILight light,
-            ICookController cooker)
```

Comment:
Is there any reason to add the power tube interface? It's already an inherent part of ICookController, which is initialized here. Adding it here also breaks the integration tests (which we aren't supposed to change) so the Jenkins job isn't building.